### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by bnadim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,21 +22,21 @@
   "license": "MIT",
   "version": "0.2.0",
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=8.9.0"
```
Comment (plouc) [inline package.json]: What's the reason for this change?
Do updated dependencies require at least this specific version?

File: package.json [hunk 2]
```diff
@@ -22,21 +22,21 @@
...
     "lodash": "4.x.x",
     "prettyoutput": "1.x.x",
     "uuid": "3.x.x"
   },
   "devDependencies": {
-    "ava": "0.21.0",
-    "coveralls": "^2.13.1",
-    "eslint": "4.4.1",
+    "ava": "1.0.0-rc.2",
+    "coveralls": "2.x.x",
```
Comment (plouc) [inline package.json]: Should be `2.x`, the same applies for other packages using this notation

File: package.json (hunk 1 above)
Comment (bnadim) [reply package.json]: i though it would be better to remove support for older version, but finally changed it back. Yet it can not be tested under node 7 and 9 (ava don't support it).
But it work on node 6.

File: package.json (hunk 2 above)
Comment (bnadim) [reply package.json]: Updated

**Comment by bnadim** [comment]
@plouc Can you give a new review please

File: package.json (hunk 1 above)
Comment (plouc) [reply package.json]: I guess it's not a big deal as they're both in _EOL_ state
